dotcom-rendering/src/web/components/Callout/Form.stories.tsx

```diff
@@ -1,60 +1,29 @@
-import { css } from '@emotion/react';
-import {
-	calloutCampaign,
-	calloutCampaignOnlyTwoRadio,
-} from '../../../../fixtures/manual/calloutCampaign';
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import { calloutCampaign } from '../../../../fixtures/manual/calloutCampaign';
 import { Form } from './Form';
 
+const mockFormat = {
+	display: ArticleDisplay.Standard,
+	design: ArticleDesign.Standard,
+	theme: ArticlePillar.Opinion,
+};
+
 export default {
 	component: Form,
 	title: 'Components/Callout/Form',
 };
 
 export const Default = () => {
 	return (
-		<div
```

Comment:
I thought having three different example stories was overkill when you could actually just have the one that captures all the possible elements, hence I removed these

Reply:
This is a good idea. This PR just adds the form that will live inside the `CalloutBlockComponent` - not the component itself. The component is going to take care of whether it's collapsed, expired etc. I'll ensure I capture all of these stories in a subsequent PR. How does that sound?